src/pluginsType.ts

```diff
@@ -0,0 +1,28 @@
+// 插件内用到的类型进行统一定义
```

Comment:
can this file be renamed to just `types.ts` ?

Reply:
I wanted to change it to type.ts, but I think my current name looks a little more intuitive

Reply:
perhaps `lib-types.ts` would be more consistent?